src/main/java/salesforce/ui/pages/BasePage.java

```diff
@@ -10,23 +10,22 @@
 
 import core.selenium.WebDriverManager;
 import core.selenium.WebElementAction;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.support.PageFactory;
-import org.openqa.selenium.support.ui.WebDriverWait;
 
 /**
  * Load home page.
  */
 public abstract class BasePage {
     public WebElementAction webElementAction;
-    public WebDriverWait wait;
-    public WebDriver driver;
+    public WebDriverManager webDriverManager;
 
     /**
-     * Initializes the elements and wait for page to be loaded.
+     * Initializes web element actions.
+     *
+     * @param webDriverManager .
```

Comment:
According to convention the param and return annotations shouldn't have a dot at the end of the phrase, rather checkstyle wants a description, so you could add a short one. Only the method description ends with a dot.